tools/configure.c

```diff
@@ -382,10 +382,18 @@ static int run_make(const char *arg)
   char **argv;
 
   snprintf(g_buffer, BUFFER_SIZE, "make %s", arg);
-  for (argv = g_makeargv; *argv; argv++)
+
+  if (!*g_makeargv)
+    {
+      strncat(g_buffer, " 1>/dev/null", BUFFER_SIZE - 1);
```

Comment:
Window native don't have /dev/null, need do something like the previous code

Reply:
Good catch. I'll fix it.